### ISSUE TITLE
Not working on iOS 8

### ISSUE THREAD

**Issue by SPDEX** [mikaelhellqvist/ReverseClip#10]
The output file imageSequence_reverse.mov has 0 seconds and when is played displays few frames and then it stops.

**Comment by amrittrivedir** [comment]
@SPDEX  have you find any solution for this?

**Comment by SPDEX** [comment]
No, I didn't manage to fix it. Any idea ?

**Comment by iampena** [comment]
I am using ReverseClip on iOS 8 and it seems to work just fine.
Did you change the original code?

**Comment by SPDEX** [comment]
Here is the result, without any changes to the source code

Original video
http://tinypic.com/r/2wmqubs/8

Reversed video
http://tinypic.com/r/208zg60/8

**Comment by amrittrivedir** [comment]
@SPDEX  Yes, i am getting same result.  I tried with static value in this file RCImageSequence while(clipTime < durationSeconds) {
        CMTime frameTime = CMTimeMakeWithSeconds(durationSeconds - clipTime, FRAME_SCALE);
        NSValue *frameTimeValue = [NSValue valueWithCMTime:frameTime];

```
    [times addObject:frameTimeValue];

    clipTime += 0.01;
};
```

Cliptime is static now it works but didn't getting desired result.  

@iampena  have you changed any sort of code? 
Anyone have idea to reverse the audio?

**Comment by vijaytholpadi** [comment]
## EDIT: Playing with the FRAME_RATE fixed this for me.

Did anyone manage to solve this?

For me, strangely it fails only on the iPhone 6 and that too not consistently. 20% of the times it goes through. Any idea guys?
